types/marked/marked-tests.ts

```diff
@@ -15,6 +15,12 @@ tokenizer.emStrong = function emStrong(src, _maskedSrc, _prevChar) {
     return token;
 };
 
+tokenizer.inlineText = function inlineText(...args: Parameters<marked.Tokenizer['inlineText']>) {
+    const p = tokenizer.inlineText(...args);
+
+    return p;
+};
+
```

Comment:
Since you are updating the `this` parameter you should probably use that in the test.

Maybe something like:

```js
tokenizer.inlineText = function inlineText(...args: Parameters<marked.Tokenizer['inlineText']>) {
    const p = tokenizer.inlineText(...args);

    if (this.options.baseUrl) {
        p.text = this.lexer.options.baseUrl;
    }

    return p;
};
```

Reply:
`text` and `baseUrl` have different types, will conflict
I will just refer to `this.inlineText` - that should already not fail as having `Tokenizer & TokenizerThis` types